Python/Product/EnvironmentsList/PipExtension.xaml.cs

```diff
@@ -76,21 +77,20 @@ internal sealed partial class PipExtension : UserControl {
         }
 
         private void UpgradePackage_CanExecute(object sender, CanExecuteRoutedEventArgs e) {
+            e.Handled = true;

```

Comment:
I'm inclined to agree, we (mostly me) have looked pretty free with what goes into 2.2.1 so far, though I'm not aware of anything significant other than crash/unhandled error fixes.

We'll get to have an RC for this one, as well as internal testing, so I'm not too concerned yet.